### ISSUE TITLE
Fix redis connection

### ISSUE THREAD

**Issue by crowesn** [crowesn/hyrax-docker#16]
`$ docker logs hyrax-docker_worker_1`

```
/scripts/wait-for.sh: line 43: redis-cli: command not found
redis is not yet available.
```

**Comment by crowesn** [comment]
```
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff
Content-Type: text/html; charset=utf-8
ETag: W/"3a57a7df2fb73b1ad3f62189159fa124"
Cache-Control: max-age=0, private, must-revalidate
Set-Cookie: _night_moves_session=RDJaaWhNeld6dXAwN2h1QWhoOFppTWczSmp2UkdlUkhmSHR4S0tieUVqeGpTeGhzNU1pRmEycytYenY0NllPbUZIMWtUOHY1WmcxWGo4TGFyYythRVR1VTdBVGE3Uit2akpVYXZzc2w3M09sYUNhd2F3U2ovczdJUWowWHlTeFU4QVJnejRuc0pFS252dmY3WUdDRjdRPT0tLXUrNXBEaHNsb3g3MTZWYmQ3UG9tcWc9PQ%3D%3D--acaa42ad8d3269111b7bae7a115c81baf0511bb9; path=/; HttpOnly
X-Request-Id: cd4d4b54-ee6a-412c-a7ff-be6fdfb44ca6
X-Runtime: 22.786182

rails is available.
Install bundler
Successfully installed bundler-2.0.2
Parsing documentation for bundler-2.0.2
Installing ri documentation for bundler-2.0.2
Done installing documentation for bundler after 3 seconds
1 gem installed
Checking and Installing Ruby Gems
The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x86-mswin32, x64-mingw32, java. To add those platforms to the bundle, run `bundle lock --add-platform x86-mingw32 x86-mswin32 x64-mingw32 java`.
The Gemfile's dependencies are satisfied
Stopping Existing sidekiq Tasks
Starting Sidekiq
bundler: command not found: sidekiq
Install missing gem executables with `bundle install`
```

**Comment by crowesn** [comment]
Resolved by #20
